scripts/encode2ga4gh.py

```diff
@@ -0,0 +1,296 @@
+#!/usr/bin/env python
+from __future__ import division
+
+import os
+import sys
+import json
+import errno
+import urllib2
+import optparse
+
+
+class Color:
+    """
+    Color printing in terminal
+    """
+
+    RED = '\033[91m'
+    BLUE = '\033[94m'
+    GREEN = '\033[92m'
+    NORMAL = '\033[0m'
+    YELLOW = '\033[93m'
+
+    @staticmethod
+    def red(astr):
+        return Color.RED + str(astr) + Color.NORMAL
+
+    @staticmethod
+    def green(astr):
+        return Color.GREEN + str(astr) + Color.NORMAL
+
+    @staticmethod
+    def yellow(astr):
+        return Color.YELLOW + str(astr) + Color.NORMAL
+
+    @staticmethod
+    def blue(astr):
+        return Color.BLUE + str(astr) + Color.NORMAL
+
+
+def log(message):
+    print >>sys.stderr,  "== " + Color.green("{}".format(message))
+
+
+def get_files_from_host(data, host, output_type, request=False, subset=None):
+    file_list = [f for f in data.get('files') if (f.get('output_type')
+                 == output_type)]
+    if subset:
+        file_list = file_list[:subset]
+    for f in file_list:
+        experiment = f.get('dataset').split('/')[-2]
+        replicate = str(f.get('replicate').get('biological_replicate_number'))
+        analysisId = "_".join((experiment, replicate))
+
+        if not request:
+            yield analysisId
+        else:
+            href = f.get('href')
+            url = "{host}{file}".format(host=host, file=href)
+            yield (analysisId, urllib2.urlopen(url))
+
+
+def get_data_from_host(url, headers, host, output_type, output_folder,
+                       description, annotationId, subset=None):
+    req = urllib2.Request(url, headers=headers)
+    try:
+        response = urllib2.urlopen(req)
+    except urllib2.URLError as e:
+        if hasattr(e, 'reason'):
+            print 'We failed to reach a server.'
+            print 'Reason: ', e.reason
+        elif hasattr(e, 'code'):
+            print 'The server couldn\'t fulfill the request.'
```

Comment:
Instead of using escapes, you can just surround the string with the quote marks you are not using:
```
"The server couldn't fulfill the request."
```